### PR TITLE
Implement durability QoS

### PR DESCRIPTION
**Pull request by pablogs9**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -29,9 +29,9 @@ endif()
 
 project(microxrcedds VERSION "2.0.0")
 
-set(_client_tag develop)
+set(_client_tag feature/implement_durability)
```
Comment (pablogs9) [inline CMakeLists.txt:32]: Before merge:

```suggestion
set(_client_tag develop)
```

File: CMakeLists.txt [hunk 2]
```diff
@@ -29,9 +29,9 @@ endif()
...
 set(_client_version 2.0.0)
-set(_agent_tag develop)
+set(_agent_tag feature/implement_durability)
```
Comment (pablogs9) [inline CMakeLists.txt:34]: Before merge:

```suggestion
set(_agent_tag develop)
```

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/200/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/219/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/201/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/220/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/202/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/221/)

**Review by Acuadros95 — APPROVED** [review]

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/204/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/223/)

**Pull request merged**